library/pk_wrap.h

```diff
@@ -53,7 +53,7 @@ struct mbedtls_pk_info_t
     /** Make signature */
     int (*sign_func)( void *ctx, mbedtls_md_type_t md_alg,
                       const unsigned char *hash, size_t hash_len,
-                      unsigned char *sig, size_t *sig_len,
+                      unsigned char *sig, size_t sig_size, size_t *sig_len,
```

Comment:
I think we should change the type `mbedtls_pk_rsa_alt_sign_func` accordingly. I hope the CI will catch this, otherwise it means we have a big test gap.

Reply:
No, I deliberately didn't change the RSA functions, neither the built-in ones nor the alt one. For RSA, the wrapper code checks the size. For ECDSA, it passes the size down to the ECDSA implementation.

`MBEDTLS_PK_RSA_ALT_SUPPORT` is enabled in `full` and tested in `test_suite_pk`.

Reply:
Oh, sorry, my mistake. I though for `PK_RSA_ALT` we called the user-provided functions directly, but that's not the case and we are wrapping them. Sorry for the noise.